GraphsV2/Components/utils/Legend/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {
+    Legend as LegendComponent
+} from 'recharts';
+
+import { LEGEND_SEPARATE } from '../../../../constants';
+import * as customLegends from './export';
+import StandardLegend from './StandardLegend';
+
+export default ({ legend, height, type, ...rest }) => {
+    let Component = type ? customLegends[type] : StandardLegend;
```

Comment:
if there is no customLegends[type] you should use StandardLegend: 
`let Component = type ? customLegends[type] || StandardLegend : StandardLegend;`